compiler/optimizer/Structure.cpp

```diff
@@ -363,8 +363,7 @@ void TR_RegionStructure::addGlobalRegisterCandidateToExits(TR_RegisterCandidate
    }
 
 
-
-static bool findCycle(TR_StructureSubGraphNode *node, TR::SparseBitVector &regionNodes, TR_BitVector &nodesSeenOnPath, TR_BitVector &nodesCleared, int32_t entryNode)
+static bool findCycle(TR_StructureSubGraphNode *node, TR::BitVector &regionNodes, TR_BitVector &nodesSeenOnPath, TR_BitVector &nodesCleared, int32_t entryNode)
```

Comment:
Maybe what should be done is moving the `typedef` such that we can use the `SparseBitVector` typedef in this signature -- (after it has been renamed of course. Perhaps `StructureBitVector`?